packages/plugins/__template/src/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview TODO: Add plugin overview.
+ */
+
+/**
+ * TODO: Rename plugin and add plugin description.
+ */
+export default class Plugin {
+  /**
+   * Constructor for ...
+   * @param {!Blockly.WorkspaceSvg} workspace The workspace that the plugin will
+   *     be added to.
+   */
+  constructor(workspace) {
+    /**
+     * The workspace.
+     * @type {!Blockly.WorkspaceSvg}
+     * @protected
+     */
+    this.workspace_ = workspace;
+  }
+
+  /**
+   * Initialize.
```

Comment:
See other comment on function comments.

Reply:
Not following?

Reply:
I guess my review comment didn't get saved. I had made a comment on another line. It was approximately:

I would like for us to follow the google style guide for wording function comments ("written in the third person, as if there is an implied This method ... before it")
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments